src/main/frontend/components/theme.cljs

```diff
@@ -65,13 +65,13 @@
      [system-theme?])
 
     (rum/use-effect!
-      #(state/set-modal!
-         (when settings-open?
-           (fn [] [:div.settings-modal (settings/settings)])))
-      [settings-open?])
+     #(state/set-modal!
+       (when settings-open?
+         (fn [] [:div.settings-modal (settings/settings)])))
+     [settings-open?])
 
     [:div
-     {:class    (str theme "-theme")
+     {:class    (util/classnames [(str theme "-theme") {:white-theme (= "light" theme)}])
```

Comment:
Hi @g1eny0ung
Is that better of keeping the `white-theme` class name could be as compatible as possible with the existing themes?

Reply:
Agree with u. I will add a comment to note that this part is for backward compatibility.

Reply:
Already fixed. PTAL again. 😺